### DISCUSSION TITLE
Use default pip instead of Poetry

### DISCUSSION THREAD

**Discussion by syrull** [SSIvanov19/a-data-pro-internship#22] [General]
https://github.com/SSIvanov19/a-data-pro-internship/blob/05d71ec1a0404efb427346694c5ef84c23274c76/Dockerfile#L12

Използвайте тук `pip install -r requirements.txt`  вместо poetry, резултатите от pip се кешират от докер докато тези от poetry не което води до бавни build-ове на docker image-a.  

За да не си слагате много работа още ползвайте тази команда за да експортнете в текст, също може да си го добавите в билда автоматично както ви е по-лесно:

`poetry export -f requirements.txt --without-hashes` 

Принципно има команди с които може да използвате и чисто poetry без pip. Ако желаете пробвайте се, тук е темата: https://github.com/python-poetry/poetry/discussions/1879

**Comment by SSIvanov19** [comment]
Направих го на discussion, защото то е предназначено за такъв вид постове

**Reply by syrull** [reply]
За да не правя нов пост разгледайте и тези 3те:

https://black.readthedocs.io/en/stable/
https://flake8.pycqa.org/en/latest/
https://pre-commit.com/


Това са стандарти за code quality и naming conventions със сигурност биха повлияли на крайния резултат.  Успех.

**Reply by SSIvanov19** [reply]
Намерих какво ще чета преди лягане 😊, благодаря за съвета!
